docs/reference/search/search.asciidoc

```diff
@@ -480,6 +480,39 @@ A boost value greater than `1.0` increases the score. A boost value between
 `0` and `1.0` decreases the score.
 ====
 
+experimental::[]
+[[search-api-knn]]
+`knn`::
+(Required, object) Defines the kNN query to run.
++
```

Comment:
Should we include a reference here to the `approximate-knn` section of the  `knn-search.asciidoc`? I was thinking if a user stumble on this page, they would not know that we have mode details on knn search.

Reply:
Good idea, I'll add a link. I also realized this should be "Optional" instead of "Required".